### PR TITLE
Feat/add english translation

### PR DESCRIPTION
**Pull request by maximakymenko**
I've managed to add `i18n` and `react-i18n` to handle internationalisation. Also I've created two separate files for PL and EN versions and for now simple function to switch between two languages.

I've translated all text strings that were in the app and tested it. Should be good :) 

Of course, this function needs some improvements to handle more than two languages.

### CODE REVIEW

File: src/components/Layout/Desktop.jsx [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
   const { useDarkTheme, setUseDarkTheme } = useTheme();
 
   return (
     <>
       <Layer>
-        <Map className={useDarkTheme ? 'dark-theme' : ''} />
+        <Map className={useDarkTheme ? "dark-theme" : ""} />
       </Layer>
       <Layer>
+<<<<<<< HEAD
```
Comment (konradkalemba) [inline src/components/Layout/Desktop.jsx]: I think you left unresolved confict here.

Comment (maximakymenko) [reply src/components/Layout/Desktop.jsx]: Thanks, I had a couple of them, so didn't notice this one. Fixed it.

**Comment by konradkalemba** [comment]
Works well! 👍 I noticed a typo tho - I think it should be `voivodeships` instead of `voivodships`.

File: src/components/DailyGrowth/DailyGrowth.jsx [hunk 2]
```diff
@@ -1,96 +1,136 @@
...
-    setGroupedData(accumulateData([...preparedCases, ...preparedDeaths, ...preparedCures]));
+    setGroupedData(
+      accumulateData([...preparedCases, ...preparedDeaths, ...preparedCures])
+    );
   }, [cases, deaths, cures]);
 
   return (
     <StyledCard
       style={$theme => ({
+        width: "320px",
```
Comment (kkostepski) [inline src/components/DailyGrowth/DailyGrowth.jsx]: By changing this part of the app, you will actually revert this PR https://github.com/konradkalemba/korona.ws/pull/41

File: src/components/DailyGrowth/DailyGrowth.jsx [hunk 3]
```diff
@@ -1,96 +1,136 @@
...
-                <Line name="Potw. przypadki" type="monotone" dataKey={CASES_KEY} strokeWidth={2} stroke={theme.colors.negative} dot={false} />
-                <Line name="Zgony" type="monotone" dataKey={DEATHS_KEY} strokeWidth={2} stroke={theme.colors.primary} dot={false} />
-                <Line name="Wyleczenia" type="monotone" dataKey={CURES_KEY} strokeWidth={2} stroke={theme.colors.positive} dot={false} />
-              </LineChart>
-            </ResponsiveContainer>
-          )}
+        {isLoading ? (
+          <Loader />
+        ) : (
+          <ResponsiveContainer height={180} width={"99%"}>
```
Comment (kkostepski) [inline src/components/DailyGrowth/DailyGrowth.jsx]: same here

**Comment by maximakymenko** [comment]
Thanks for noticing, I’ll fix it later today

**Comment by konradkalemba** [comment]
> To be honest I don't like idea of reformatting code "by the way". Most of changes are related to code reformat. I can see that the code is now "prettier", but It caused pull request is very hard to read :) But final decision/opinion is up to owner @konradkalemba

Yes, @kkostepski is right. I didn't notice the formatting changes at first, but you can leave them for now. I'll configure `prettier` tonight to have consistent code formatting across all components.

Make changes to ensure your PR won't reverse any other PRs that @kkostepski pointed out.

**Comment by maximakymenko** [comment]
Sorry guys, it happened  because of conflicts. @kkostepski thanks a lot for pointing that out 👍 I've revert changes from last PR and fixed the typo.

**Comment by konradkalemba** [comment]
Hey, thanks for fixes. I found two more mistakes:
image
image

**Comment by maximakymenko** [comment]
Thanks, didn’t notice as well. Will be fixed soon ;)

File: src/components/Layout/Desktop.jsx [hunk 4]
```diff
@@ -25,21 +27,14 @@ export default function Layout() {
         <Map className={useDarkTheme ? "dark-theme" : ""} />
       </Layer>
       <Layer>
-        <Block position={'fixed'} bottom={0} left={0} width={['100%', '100%', 'auto']} margin={['0', '0', '20px']}>
-          <Block
```
Comment (kkostepski) [inline src/components/Layout/Desktop.jsx]: This part of code (entire <Block> with override styles should be included as well :P

**Comment by maximakymenko** [comment]
Alright guys, now it should be good! I fixed all naming typos and reverted the last changes for responsiveness. 

Thanks again for all your remarks :) and good night! 

@kkostepski  @konradkalemba  thank you guys!

**Comment by konradkalemba** [comment]
@maximakymenko Thanks for your contribution. I appreciate it. :)

**Comment by kkostepski** [comment]
I have tested it as well. Good job @maximakymenko!

**Pull request merged**